### PR TITLE
added move/remove functionality and score

### PR DESCRIPTION
**Pull request by barrieca**

### CODE REVIEW

**Comment by edwardrh** [comment]
Code looks good and definitely beneficially with the view and back end of the page/game overall.

File: Assignment2/src/main/java/assets/css/AcesUp.css [hunk 1]
```diff
@@ -3,7 +3,7 @@ width: 20%;
 }
 
 td{
-width:114px;
+width:120px;
```
Comment (parkj6) [inline Assignment2/src/main/java/assets/css/AcesUp.css:6]: why did you change it back to 120px?
